src/D2L.CodeStyle.Analyzers/Immutability/ImmutabilityAnalyzer.cs

```diff
@@ -325,60 +325,57 @@ CancellationToken cancellationToken
 		}
 
 		private static void AnalyzeConflictingImmutabilityOnTypeParameters(

```

Comment:
I think this one reads better now, and it being on only namedtype type parameters is good.